src/website/app/pages/Color/ColorRampRow.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import colorable from 'colorable';
+import readableColor from 'polished/lib/color/readableColor';
+import { createStyledComponent, color } from '../../../../utils';
+
+type Props = {
+  name: string
+};
+
+const styles = {
+  row: ({ theme, color }) => ({
+    backgroundColor: color,
+    display: 'flex',
+    fontSize: theme.fontSize_mouse,
+    justifyContent: 'space-between',
+    listStyle: 'none',
+    padding: theme.spacing_double,
+    width: '100%',
+
+    '& > div > p:first-child': {
+      margin: `0 0 ${theme.spacing_single} 0`
+    },
+    '& > div > p:last-child': {
+      margin: 0
+    }
```

Comment:
If we have control over the markup, we should avoid use of descendant selectors.